com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateServiceStep.java

```diff
@@ -0,0 +1,71 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudServiceExtended;
+import com.sap.cloud.lm.sl.cf.core.cf.clients.ServiceInstanceGetter;
+import com.sap.cloud.lm.sl.cf.core.cf.clients.ServiceWithAlternativesCreator;
+import com.sap.cloud.lm.sl.cf.core.cf.services.ServiceOperationType;
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+
+@Component("createServiceStep")
+public class CreateServiceStep extends AsyncFlowableStep {
+
+    @Inject
+    private ServiceWithAlternativesCreator.Factory serviceCreatorFactory;
+
+    @Inject
+    private ServiceInstanceGetter serviceInstanceGetter;
+
+    @Override
+    protected StepPhase executeAsyncStep(ExecutionWrapper execution) throws Exception {
+        CloudServiceExtended serviceToCreate = StepsUtil.getServiceToProcess(execution.getContext());
+        ResponseEntity<String> createServiceResponse = createService(execution.getContext(), execution.getControllerClient(),
+            serviceToCreate);
+        if (HttpStatus.ACCEPTED.equals(createServiceResponse.getStatusCode())) {
+
+            Map<String, ServiceOperationType> serviceOperation = new HashMap<>();
+            serviceOperation.put(serviceToCreate.getName(), ServiceOperationType.CREATE);
```

Comment:
This is the polling operation that was used before the Create or Update refactoring. It was not part of the current change, because it involves creating a separate polling abstraction to accommodate the need of the many new polling operations.